src/Roslyn.Suppressions/Roslyn.Suppressions/Roslyn.Suppressions.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Please include the comment that was in the prototype. It will help with maintenance in the future.

```
  <!-- NOTE: this assembly should NOT have any dependencies on Visual Studio assemblies
       
	   It could be loaded into the external Roslyn analysis process by VS, so it has a
	   number of limitations about what it can reference / access
	   
	   * it can't use MEF to import any of our types.
        * it can't access any instances/services running in the main VS instance.
 
  -->
```